api/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM openjdk:13
 WORKDIR /usr/src/ab2d-api
 ADD target /usr/src/ab2d-api
-CMD java -jar api-*-SNAPSHOT.jar
+CMD java -Dspring.profiles.active=${SPRING_PROFILE} -jar api-*-SNAPSHOT.jar
```

Comment:
Based on the comments I have left so far, we might not even need the `e2d` application.properties file, which essentially eliminates the need for profiles.